include/verilated.h

```diff
@@ -397,11 +371,7 @@ class VerilatedContext VL_NOT_FINAL {
     // Implementation details
     const std::unique_ptr<VerilatedContextImpData> m_impdatap;
     // Number of threads to use for simulation (size of m_threadPool + 1 for main thread)
-#ifdef VL_THREADED
     unsigned m_threads = std::thread::hardware_concurrency();
-#else
-    const unsigned m_threads = 1;
-#endif
```

Comment:
I'm unsure about this change. I think it changes default behavior and if user doesn't set threads manually in ``cpp`` file it sets them to ``std::thread::hardware_concurrency()`` even when he set ``--threads 1`` in verilator command line. 

Maybe it should be ``1`` is a default value?

Reply:
Ok, thanks for the explanation.